build.sh

```diff
@@ -1,4 +1,2 @@
 set -e
-packer build -except publish pipeline/packer/inbound.json
-packer build -except publish pipeline/packer/outbound.json
-packer build -except publish pipeline/packer/spineroutelookup.json
+docker build -t local/sds:${BUILD_TAG} -f sds/spine-route-lookup/Dockerfile .
```

Comment:
Horrible one for you :) Let's rename spine-route-lookup folder (and sds) to spine-directory-service. This is potentially the starting for an API that does more than just spine route lookup